ci/checks/enforce-docs-up-to-date.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+pip install bs4
```

Comment:
Pls extract to a dedicated requirements.*.txt with fixed versions (as I've done with all other dependencies recently)